Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -380,8 +380,10 @@ namespace pixelTopology {
 
     static constexpr uint16_t numberOfModules = 3892;
 
-    // 1024 bins, 10 bits
-    static constexpr uint16_t clusterBinning = 1024;
+    // 1000 bins (<10 bits) must be:
```

Comment:
I think you mean that the number of bins has to fit in 10 bins, so maybe better
```suggestion
    // 1000 bins < 1024 bins (10 bits) must be:
```